src/backend/db/products.js

```diff
@@ -7,24 +7,142 @@ import { v4 as uuid } from "uuid";
 
 export const products = [
   {
-    _id: uuid(),
-    title: "You Can WIN",
-    author: "Shiv Khera",
-    price: "5000",
-    categoryName: "non-fiction",
+    _id: "b481bf23-8de2-4126-8f95-9ba35ec1b8bc",
+    title: "Women Ethnic Wear",
+    desc: "If you love flaunting ethnic print kurtas, then this whole set is for you. It is a set of two, a straight kurta with trousers. Crafted with pure cotton and pastel colour makes it soft to touch and apt to wear in any season.",
+    price: "500",
+    sell_price: "300",
+    rating: "4",
+    imageUrl: "https://i.ibb.co/qBNTfbw/Women-Ethnic-Wear.webp",
+    categoryName: "Women",
+    newArrival: true,
+  },
+
+  {
+    _id: "dc77fe75-8a72-4e28-8cab-7e464665cd89",
+    title: "Women's Jogger",
+    desc: "Peach medium wash 5-pocket high-rise cropped jeans, clean look, light fade, has a button and zip closure, and waistband with belt loops",
+    price: "900",
+    sell_price: "750",
+    rating: "3",
+    imageUrl: "https://i.ibb.co/BP021tV/Women-1.webp",
+    categoryName: "Women",
+    trending: true,
   },
   {
-    _id: uuid(),
-    title: "You are Winner",
-    author: "Junaid Qureshi",
-    price: "3000",
-    categoryName: "horror",
+    _id: "db8a4662-46e0-4a16-a502-fb81c8566d6a",
+    title: "Men's Sweater",
+    desc: "Navy Blue solid sweater, has a round neck, long sleeves, and ribbed hem",
+    price: "1100",
+    sell_price: "1050",
+    rating: "5",
+    imageUrl: "https://i.ibb.co/26KFzDB/Winter-2.webp",
+    categoryName: "Winter",
   },
   {
-    _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
+    _id: "f64aae9a-4488-42c9-ba32-544ac2b51d13",
+    title: "Women's Denim Jacket",
+    desc: "Blue floral printed denim jacket, has a spread collar,2 flap pockets, has a button closure, long sleeves, straight hemline, without lining",
     price: "1000",
-    categoryName: "fiction",
+    sell_price: "850",
+    rating: "2",
+    imageUrl: "https://i.ibb.co/7kVtpGd/Winter-1.webp",
+    categoryName: "Winter",
+  },
+  {
+    _id: "cdf0addb-36d7-4fdd-b538-55c86cd01d3a",
+    title: "Men's T-Shirt",
+    desc: "White solid T-shirt, has a round neck, and short sleeves",
+    price: "500",
+    sell_price: "450",
+    rating: "1",
+    imageUrl: "https://i.ibb.co/ryTb9gy/Summer-2.webp",
```

Comment:
I would suggest to host images on cloudinary in case thery get removed from the original website